pgdb/queries.md

```diff
@@ -0,0 +1,46 @@
+### Test Data for Blocks, Vins, Tickets and Transactions tables in Dcrdata
+
+#### The Postgresql db dump was generated from testnet data as below:
+
+- Login into the Postgres CLI shell by:
+```Shell
+    psql -U postgres
+```
+
+- While in the psql shell connect to the dcrdata testnet db (or any other source data db of your choice).
+```Shell
+    \c testnet_dcrdata
+```
+- Executes the following commands while in the postgres CLI shell. The commands will create new tables with the required data.
+
+```Shell
+    CREATE TABLE test_addresses AS SELECT * FROM addresses WHERE block_time > '2019-02-17 03:23:23+03' LIMIT 200;
```

Comment:
The **WHERE** condition(s) and **LIMIT** value can be changed to ones preference.
The custom scanner that reads the generated *.sql dump, drops the **test_** prefix from the tables automatically